dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -258,3 +258,105 @@ CWWKD1023.extra.param.explanation=Query conditions that define \
  parameters to supply the values. Extra parameters are not permitted.
 CWWKD1023.extra.param.useraction=Update the repository method signature \
  to remove extra parameters that do not correspond to declared conditions.
+
+CWWKD1024.missing.entity.prop=CWWKD1024E: The {0} method of the {1} repository \
+ interface is missing an entity property name or was supplied with a parameter \
+ that lacks an entity property name. Valid property names for the {2} entity \
+ class are: {3}.
+CWWKD1024.missing.entity.prop.explanation=The query conditions and sort criteria \
+ for a repository method must include an entity property name.
+CWWKD1024.missing.entity.prop.useraction=Inspect the declaration and usage of \
+ the repository method to ensure that an entity property name is supplied for \
+ each query condition and sort criterion.
+
+CWWKD1025.missing.id.prop=CWWKD1025E: The {0} entity class that is used by the \
+ {1} method of the {2} repository interface does not have an Id property or \
+ accessor method.
+CWWKD1025.missing.id.prop.explanation=Repository methods that delete and return \
+ an entity must have an Id property.
+CWWKD1025.missing.id.prop.useraction=If the entity does not have a single \
+ property that is the Id, define separate repository methods to query for the \
+ entity and delete the entity.
+
+CWWKD1026.ignore.case.not.text=CWWKD1026E: The {0} property of the {1} entity \
+ is not a character or character sequence, so the {2} sort criteria must not \
+ specify to ignore case. The type of the property is {3}, and it was supplied to \
+ the {4} method of the {5} repository interface.
+CWWKD1026.ignore.case.not.text.explanation=Sorting with case ignored is only \
+ possible with entity properties that represent text.
+CWWKD1026.ignore.case.not.text.useraction=Ensure the sort criteria is applied \
+ to the correct entity property and remove the instruction to ignore case if the \
+ entity property does not represent text.
+
+CWWKD1027.anno.missing.prop.name=CWWKD1027E: Parameter {0} of the {1} method of \
+ the {2} repository interface is annotated with an {3} annotation that does not \
+ specify an entity property name as its value. The annotation must either specify \
+ the entity property name as its value or the -parameters compile option must be \
```

Comment:
change
value
to
value,

Reply:
Thanks, this makes it more clear. Updating as recommended.